<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WKSEDG-MJ05N2PK" name="executableClass.Fully_Enrolled_Customers_Flow" tests="1" failures="1" timestamp="18 Feb 2019 05:58:04 GMT" time="1.157" errors="0">
  <testcase name="TC003_EnrollNewProgram_Phonenumber_VerifybyPII" time="1.157" classname="executableClass.Fully_Enrolled_Customers_Flow">
    <failure type="java.lang.AssertionError" message="Last updated date in cust alias table is not changed in DB">
      <![CDATA[java.lang.AssertionError: Last updated date in cust alias table is not changed in DB
	at org.testng.Assert.fail(Assert.java:94)
	at computedClass.Computed_DB.DB_CheckEnrolledDetails(Computed_DB.java:356)
	at executableClass.Fully_Enrolled_Customers_Flow.TC003_EnrollNewProgram_Phonenumber_VerifybyPII(Fully_Enrolled_Customers_Flow.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- TC003_EnrollNewProgram_Phonenumber_VerifybyPII -->
</testsuite> <!-- executableClass.Fully_Enrolled_Customers_Flow -->
